packages/patterns/src/patterns/getGuardPayloads.js

```diff
@@ -203,10 +203,6 @@ const LegacyInterfaceGuardShape = M.splitRecord(
   {
     defaultGuards: M.or(M.undefined(), 'passable', 'raw'),
     sloppy: M.boolean(),
-    // There is no need to accommodate LegacyMethodGuardShape in
-    // this position, since `symbolMethodGuards happened
-    // after https://github.com/endojs/endo/pull/1712
-    symbolMethodGuards: M.mapOf(M.symbol(), MethodGuardShape),
```

Comment:
Need to figure out how to retire this with some kind of deprecation dance instead.